include/sys/vnode.h

```diff
@@ -24,6 +24,10 @@ typedef struct cred cred_t;
  * vnodeops should not modify attributes set to VNOVAL. */
 #define VNOVAL (-1)
 
+/* Flag for vattr:va_flags.
+ * Utimes argument was null. */
+#define VA_UTIMES_NULL 1
```

Comment:
I think that we prefer `enum` for flags.

Reply:
Changed.